firebase/tools/collections-diff.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Not sure about this script folder. It is a script but it's not related with the application domain, it's related to the CI environment that we are using.

Reply:
I think it is related to the application. Instead of passing the arguments directly to the sync use-case, maybe it could request to some gateway that knows how to make a diff? Not sure, there are a couple of ways to implement this, but just like we are accessing file system with the gateway, using the github API is another gateway. Just because we are consuming it through a CLI, it doesn't mean it's not a gateway. I would look into fetching it with a HTTP though, cause running CLI commands through a node application could be security-error-prone, which could be catastrophic, although if well built, I would say fine.